src/server/src/main/java/com/spotify/reaper/resources/view/NodesStatus.java

```diff
@@ -28,34 +29,53 @@
+  private static List<Pattern> endpointDcPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointRackPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointLoadPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointReleasePatterns = Lists.newArrayList();
+  private static List<Pattern> endpointSeverityPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointHostIdPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointTokensPatterns = Lists.newArrayList();
+
   private static Pattern endpointNamePattern = Pattern.compile("^([0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3})", Pattern.MULTILINE | Pattern.DOTALL);
-  private static Pattern endpointStatusPattern = Pattern.compile("(STATUS):([0-9]*):(\\w+)");
-  private static Pattern endpointDcPattern = Pattern.compile("(DC):([0-9]*):([0-9a-zA-Z-\\.]+)");
-  private static Pattern endpointRackPattern = Pattern.compile("(RACK):([0-9]*):([0-9a-zA-Z-\\.]+)");
-  private static Pattern endpointLoadPattern = Pattern.compile("(LOAD):([0-9]*):([0-9eE.]+)");
-  private static Pattern endpointReleasePattern = Pattern.compile("(RELEASE_VERSION):([0-9]*):([0-9.]+)");
-  private static Pattern endpointSeverityPattern = Pattern.compile("(SEVERITY):([0-9]*):([0-9.]+)");
-  private static Pattern endpointHostIdPattern = Pattern.compile("(HOST_ID):([0-9]*):([0-9a-z-]+)");
-  private static Pattern endpointTokensPattern = Pattern.compile("(TOKENS):([0-9]*)");
-  
+  private static Pattern endpointStatus22Pattern = Pattern.compile("(STATUS):([0-9]*):(\\w+)");
+  private static Pattern endpointDc22Pattern = Pattern.compile("(DC):([0-9]*):([0-9a-zA-Z-\\.]+)");
+  private static Pattern endpointRack22Pattern = Pattern.compile("(RACK):([0-9]*):([0-9a-zA-Z-\\.]+)");
+  private static Pattern endpointLoad22Pattern = Pattern.compile("(LOAD):([0-9]*):([0-9eE.]+)");
+  private static Pattern endpointRelease22Pattern = Pattern.compile("(RELEASE_VERSION):([0-9]*):([0-9.]+)");
+  private static Pattern endpointSeverity22Pattern = Pattern.compile("(SEVERITY):([0-9]*):([0-9.]+)");
+  private static Pattern endpointHostId22Pattern = Pattern.compile("(HOST_ID):([0-9]*):([0-9a-z-]+)");
+  private static Pattern endpointTokens22Pattern = Pattern.compile("(TOKENS):([0-9]*)");
+  private static Pattern endpointStatus21Pattern = Pattern.compile("(STATUS)(:)(\\w+)");
+  private static Pattern endpointDc21Pattern = Pattern.compile("(DC)(:)([0-9a-zA-Z-\\.]+)");
+  private static Pattern endpointRack21Pattern = Pattern.compile("(RACK)(:)([0-9a-zA-Z-\\.]+)");
+  private static Pattern endpointLoad21Pattern = Pattern.compile("(LOAD)(:)([0-9eE.]+)");
+  private static Pattern endpointRelease21Pattern = Pattern.compile("(RELEASE_VERSION)(:)([0-9.]+)");
+  private static Pattern endpointSeverity21Pattern = Pattern.compile("(SEVERITY)(:)([0-9.]+)");
+  private static Pattern endpointHostId21Pattern = Pattern.compile("(HOST_ID)(:)([0-9a-z-]+)");
```

Comment:
nit-pick: all these `private static` could be `private static final`. That would also means the names should be uppercase.